### PR TITLE
Replace placeholders in the README

### PR DESCRIPTION
**Pull request by eherot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 
-# Requirements
+# Recipes
 
-* `apt` cookbook
-* `some` cookbook
-* `another` cookbook
+## default
 
+Just a wrapper. Installs Vault and Consul client
```
Comment (jeffbyrnes) [inline README.md]: s/Consul client/Consul Agent/

**Comment by jeffbyrnes** [comment]
Looks good other than my one nit.

File: README.md (hunk 1 above)
Comment (eherot) [inline README.md]: [orly?](https://github.com/evertrue/et_secrets-cookbook/blob/master/recipes/default.rb#L8)

Comment (jeffbyrnes) [inline README.md]: Valid criticism. Client it is!

**Pull request merged**